test/unit/presenters/publishing_api_presenters/topical_event_about_page_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class PublishingApiPresenters::TopicalEventAboutPageTest < ActiveSupport::TestCase
+  def present(record)
+    PublishingApiPresenters::TopicalEventAboutPage.new(record)
+  end
+
+  test "topical event about page presentation includes the correct values" do
+    topical_event_about_page = create(:topical_event_about_page, content_id: SecureRandom.uuid)
```

Comment:
Setting the `content_id` should happen for free if you bring in @gpeng's recent work in #2447.